opendrift_leeway_webgui/leeway/utils.py

```diff
@@ -170,7 +170,11 @@ def mail_result_text(simulation):
     """
     Create a result mail text
     """
-    text = "Find the image attached." if simulation.img else "The simulation failed."
+    text = (
+        "Find the image attached."
+        if simulation.img
+        else f"The simulation failed:\n\n{simulation.error}"
+    )
```

Comment:
I'm not sure how technical these error messages get. Nevertheless maybe it's better to provide technical feedback than no feedback at all? If they don't understand the message, they can at least search for it on google or forward the email to somebody who might know what's the problem...

Reply:
Agreed :)